### ISSUE TITLE
add user through email generate error

### ISSUE THREAD

**Issue by patricechataigner** [the-deep/client#626]
Hi guys! i received this by email
![image](https://user-images.githubusercontent.com/15996586/51391188-b2493380-1b31-11e9-9556-5d69bf32ae66.png)

and when i click on add user ( the two buttons could be better separated i think in the email display) then it opens this window in my browser:

![image](https://user-images.githubusercontent.com/15996586/51391220-cd1ba800-1b31-11e9-91ac-1ad4c1b691b6.png)

**Comment by eoglethorpe** [comment]
hey the email formatting is better, idk why yours looks like that? could this be for the nightly env?

**Comment by eoglethorpe** [comment]
yeah really not sure how you got this patrice, it works fine for me
